doc/nrf/device_guides/working_with_fem.rst

```diff
@@ -15,80 +15,79 @@ Some FEMs, like the nRF21540, also provide a power down (PDN) control that power
 * :ref:`ug_radio_fem_skyworks` - For the implementation that uses a 2-pin interface with the SKY66112-11 device and other compatible front-end modules.
 
-To use these implementations, your application must use a protocol driver that enables the FEM feature.
+To use any of these implementations with your application, the following conditions must be satisfied:
 
-The library provides multiprotocol support, but you can also use it in applications that require only one protocol.
-To avoid conflicts, check the protocol documentation to see if the protocol uses the FEM support provided by MPSL.
+1. The :ref:`nrfxlib:mpsl_fem` in the :ref:`nrfxlib:mpsl` (MPSL) library must be enabled.
+It provides implementations of drivers for supported Front-end modules.
 
-.. note::
-   Currently, the following protocols use the FEM support provided by MPSL:
+   * Enable support for MPSL by setting the :kconfig:option:`CONFIG_MPSL` Kconfig option to ``y``. For radio protocol drivers based on MPSL, this option is selected by default.
+   * Enable support for the FEM subsystem by setting the :kconfig:option:`CONFIG_MPSL_FEM` Kconfig option to ``y``.
+     This option is selected automatically if a supported FEM is provided in the application's devicetree file.
+
+If the application cannot use MPSL but wishes to use only the FEM driver provided by MPSL, refer to :ref:`ug_radio_fem_direct_support` for details.
+#. The FEM driver described above must be used by the radio protocol driver.
+Currently, the following protocols use the FEM support provided by MPSL:
 
    * :ref:`ug_thread`
    * :ref:`ug_zigbee`
    * :ref:`ug_ble_controller`
    * :ref:`ug_multiprotocol_support`
 
-.. _ug_radio_fem_requirements:
-
-Enabling FEM and MPSL
----------------------
+For applications based on the protocols above, the FEM driver provided by MPSL is used to correctly control the FEM depending on the current radio operation.
+For other radio protocol implementations and applications that control the radio directly, the FEM driver must be used according to :ref:`nrfxlib:mpsl_api_fem`.
+For reference, see the following samples that are not based on any of the radio protocol drivers listed above and support FEM control:
 
-Before you add the devicetree node in your application, complete the following steps:
+   * :ref:`radio_test`
+   * :ref:`direct_test_mode`
 
-1. Add support for the MPSL library in your application.
-   The MPSL library provides API to configure FEM.
-   See :ref:`nrfxlib:mpsl_lib` in the nrfxlib documentation for details.
-#. Enable support for MPSL implementation in |NCS| by setting the :kconfig:option:`CONFIG_MPSL` Kconfig option to ``y``.
-#. Enable support for the FEM subsystem in |NCS| by setting the :kconfig:option:`CONFIG_MPSL_FEM` Kconfig option to ``y``.
-#. Choose the used FEM implementation by selecting the appropriate Kconfig option.
+#. The FEM must be defined in devicetree file of the application.
```

Comment:
The numbering does not work. In the doc preview build, there is a literal `#` here.